pcdsdevices/ccm.py

```diff
@@ -773,6 +773,41 @@ def inverse(self, real_pos: namedtuple) -> namedtuple:
         return self.PseudoPosition(energy=energy)
 
 
+class CCMEnergyWithACRStatus(CCMEnergyWithVernier):
+    """
+    CCM energy motor and ACR beam energy request with status.
+    Note that in this cae vernier indicates any ways that ACR will act on the
+    photon energy request. This includes the Vernier, but can also lead to
+    motion of the undulators or the K.
+
+    Parameters
+    ----------
+    prefix : str
+        The PV prefix of the Alio motor, e.g. XPP:MON:MPZ:07A
+    hutch : str, optional
+        The hutch we're in. This informs us as to which vernier
+        PVs to write to. If omitted, we can guess this from the
+        prefix.
+    acr_status_sufix : str
+        Prefix to the SIOC PV that ACR uses to report the move status.
+        For HXR this usually is 'AO805'.
+    """
+    vernier = FCpt(BeamEnergyRequest, '{hutch}',
+                   acr_status_suffix='AO805',
+                   kind='normal',
+                   doc='Requests ACR to move the energy.')
+
+    def __init__(
+        self,
+        prefix: str,
+        hutch: typing.Optional[str] = None,
```

Comment:
As is, `hutch` is ignored and not passed to `.super().__init__`.  This will work since the parent class looks at the prefix, but nothing will happen if you provide `hutch`